### ISSUE TITLE
MIgrateController: Failed to save submission “” blank message on errors

### ISSUE THREAD

**Issue by jamesmacwhite** [verbb/formie#1158]
### Describe the bug

When using the MigrateController (in my case Sprout Forms) and it encounters a submission which cannot be migrated, the message is:

```
Failed to save submission “”
exampleField: Error message
```

The `""` suggests a variable is being referenced which isn't present, looking at the migration code it would appear to be `$event->submission->id` has no value when errors occur.

For easier debugging I don't know if it would be possible to report the original elementId value or some form of property of the original submission data that failed to be migrated, if it's available.

### Steps to reproduce

1. Run the Sprout Forms migration
2. Encounter a submission in a form which cannot be migrated to Formie due to an error.
3. Note the error message provided does not provide an ID as `$event->submission->id` has no value.

### Form settings

N/A

### Craft CMS version

3.7.59

### Plugin version

1.6.16

### Multi-site?

No

**Comment by engram-design** [comment]
Improved for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.17](https://github.com/verbb/formie/releases/tag/1.6.17)

**Comment by jamesmacwhite** [comment]
Awesome! Thank you.
